packages/@lwc/engine/dom/src/apis/get-component-constructor.ts

```diff
@@ -6,8 +6,7 @@
  */
 
 import { isUndefined } from '@lwc/shared';
-import { LightningElement } from '../../src';
-import { getAssociatedVMIfPresent } from '../../../src/framework/vm';
+import { LightningElement, getAssociatedVMIfPresent } from '../../../src';
```

Comment:
these still looks fishy to me... getAssociatedVMIfPresent seems to come from core pkg, but it is used by dom.